test/test_files/ddl/concurrency.test

```diff
@@ -23,3 +21,148 @@
 ---- 1
 8
 -END_CONCURRENT_EXECUTION
+
+-CASE ConcurrentSingleWriteAlter
+-CREATE_CONNECTION conn2
+-BEGIN_CONCURRENT_EXECUTION
+-STATEMENT ALTER TABLE person ADD propA INT64;
+---- ok
+-STATEMENT [conn2] MATCH (:person) return count(*);
+---- ok
+-END_CONCURRENT_EXECUTION
+-STATEMENT CALL table_info('person') return *;
+---- 17
+0|ID|INT64|True
+1|fName|STRING|False
+2|gender|INT64|False
+3|isStudent|BOOL|False
+4|isWorker|BOOL|False
+5|age|INT64|False
+6|eyeSight|DOUBLE|False
+7|birthdate|DATE|False
+8|registerTime|TIMESTAMP|False
+9|lastJobDuration|INTERVAL|False
+10|workedHours|INT64[]|False
+11|usedNames|STRING[]|False
+12|courseScoresPerTerm|INT64[][]|False
+13|grades|INT64[4]|False
+14|height|FLOAT|False
+15|u|UUID|False
+16|propA|INT64|False
+
+-CASE ConcurrentSingleWriteCreate
```

Comment:
ConcurrentSingleWriteCreate, ConcurrentSingleWriteUpdate and ConcurrentSingleWriteDelete don't really belong to ddl, can u reorganize the tests a bit?
I would suggest move the ddl part to `transaction/ddl/` as `ddl_concurrency.test`. and separate the above three into a new test file under `transaction` as `dml_concurrency.test`.